docs/UsersGuide.asciidoc

```diff
@@ -554,6 +554,19 @@ Steps 4. to 6. can be repeated for further +assert_screen+s/needles without rest
 test.
 
 
+=== Job group editor gh#2111 ===
+
+Scenarios are defined as part of a job group. The `Edit job group` button exposes the editor.
+
+By default the scenarios are listed in a table per medium, with priority and machine(s).
+Values are changed on the fly. Scenarios are removed by removing all of their machines.
+The `Edit YAML` button can be used to reveal the YAML editor and migrate a group. After
+saving for the first time, the group will only be managed in YAML.
```

Comment:
s/will only be managed in YAML/can only be configured in YAML anymore/. I would suggest on top: "It is suggested to create a backup before migrating a group to YAML format" :)

Reply:
What kind of backup do you have in mind? Unless it's your own instance you probably can't backup or restore.
Also, we don't suggest a backup when you modify the table view, which makes accidental changes very easy imho, whereas the event even has a diff when you update the YAML.

Reply:
well for any accidential changes in the table view the user has the possibility to recover, for saving as yaml and therefore disabling the table view for good that is not easily possible. So the very least the "operator" can do is to call `dump-templates` to catch the complete state before the switch to yaml